app/src/androidTest/java/com/android/sample/ui/activityDetails/CommentItemTest.kt

```diff
@@ -30,12 +31,14 @@ class CommentItemTest {
           profileId = testProfileId,
           comment = testComment,
           onReplyComment = { _, _ -> },
-          onDeleteComment = {})
```

Comment:
Adding a creatorId parameter to CommentItem and related test cases to identify the comment's creator.